### PR TITLE
GIS Cache fixes

### PR DESCRIPTION
**Pull request by dilallkx**
**CHANGES:**
- Changed GIS Cache database scope from global to sandbox-specific
- Added 'sbox' to redis data key root

**TESTING:**
- Cypress & UT pass
- Manual testing of MEC services with multiple users

### CODE REVIEW

File: go-packages/meep-net-char-mgr/algo-segment.go [hunk 1]
```diff
@@ -387,15 +387,15 @@ func (algo *SegmentAlgorithm) createMetricsEntry(srcElem string, dstElem string)
 	creationTime["creationTime"] = time.Now()
 
 	// Entries are created with no values, sidecar will only fill them, otherwise, won't be cleared
-	_ = algo.rc.SetEntry(algo.BaseKey+dstElem+":"+srcElem, creationTime)
+	// _ = algo.rc.SetEntry(algo.BaseKey+dstElem+":"+srcElem, creationTime)
```
Comment (pastorsx) [inline go-packages/meep-net-char-mgr/algo-segment.go:390]: why commenting the creation of the entries?

File: go-packages/meep-net-char-mgr/algo-segment.go [hunk 2]
```diff
@@ -387,15 +387,15 @@ func (algo *SegmentAlgorithm) createMetricsEntry(srcElem string, dstElem string)
...
 	_ = algo.rc.SetEntry(algo.BaseKey+dstElem+":throughput", creationTime)
 }
 
 // deleteMetricsEntries -
 func (algo *SegmentAlgorithm) deleteMetricsEntries() {
 	for _, flow := range algo.FlowMap {
 		// Entries are created with no values, sidecar will only fill them, otherwise, won't be cleared
-		_ = algo.rc.DelEntry(algo.BaseKey + flow.DstNetElem + ":" + flow.SrcNetElem)
+		// _ = algo.rc.DelEntry(algo.BaseKey + flow.DstNetElem + ":" + flow.SrcNetElem)
```
Comment (pastorsx) [inline go-packages/meep-net-char-mgr/algo-segment.go:398]: similar comment as above, why commenting the code from now on?

File: go-packages/meep-net-char-mgr/algo-segment.go (hunk 1 above)
Comment (dilallkx) [reply go-packages/meep-net-char-mgr/algo-segment.go:390]: TC sidecars now use the metric-store package to set these entries in the redis DB. The net char manager no longer needs to create the entries for the sidecars.

File: go-packages/meep-net-char-mgr/algo-segment.go (hunk 2 above)
Comment (dilallkx) [reply go-packages/meep-net-char-mgr/algo-segment.go:398]: TC sidecars now use the metric-store package to set these entries in the redis DB. The net char manager no longer needs to create the entries for the sidecars.

File: go-packages/meep-net-char-mgr/algo-segment.go (hunk 1 above)
Comment (dilallkx) [reply go-packages/meep-net-char-mgr/algo-segment.go:390]: I will remove the commented line.

File: go-packages/meep-net-char-mgr/algo-segment.go (hunk 2 above)
Comment (dilallkx) [reply go-packages/meep-net-char-mgr/algo-segment.go:398]: I will remove the commented line.

**Review by pastorsx — APPROVED** [review]

**Pull request merged**